code.gs

```diff
@@ -32,14 +32,14 @@ var myEmail = 'insertyouremailhere@someemail.com';
  * If you leave it blank it will default to your Google account name.
  * Note: this may not work when using a Gmail address sending emails to itself.
  */
-var emailSenderName = 'Birthday Notifications';
+var emailSenderName = 'Contacts Events Notifications';
 
 /*
- * ID OF THE BIRTHDAY CALENDAR
+ * ID OF THE CONTACTS EVENTS CALENDAR
  *
- * Open up https://calendar.google.com, in the menu on the left click on the arrow next to the birthday calendar
```

Comment:
Mine is presently set to "Birthdays & Anniversaries" but I am almost sure I've edited that name at some point, and Google seems to have no way to see the default pre-edited name. Are you able to temporarily change your settings to English to see what the default name becomes? Otherwise we should find another user with language set to English who knows they haven't edited the calendar name yet.